frictionless_ckan_mapper/frictionless_to_ckan.py

```diff
@@ -112,19 +112,33 @@ def package(fddict):
         del outdict['keywords']
 
     final_dict = dict(outdict)
-    for key, value in outdict.items():
+    for pkey, pvalue in outdict.items():  # "package key", "package value"
         if (
-            key not in ckan_package_keys and
-            key not in frictionless_package_keys_to_exclude
+            pkey not in ckan_package_keys and
+            pkey not in frictionless_package_keys_to_exclude
         ):
-            if isinstance(value, (dict, list)):
-                value = json.dumps(value)
+            if isinstance(pvalue, (dict, list)):
+                pvalue = json.dumps(pvalue)
             if not final_dict.get('extras'):
                 final_dict['extras'] = []
             final_dict['extras'].append(
-                {'key': key, 'value': value}
+                {'key': pkey, 'value': pvalue}
             )
-            del final_dict[key]
-    outdict = dict(final_dict)
-
+            del final_dict[pkey]
+        elif pkey == 'resources':
+            # Remap differences from Frictionless to CKAN resource.
+            # List of resources as `dict`: Iterate over each resource.
+            for num, rdict in enumerate(outdict[pkey]):  # resource dict
```

Comment:
why are we doing stuff to resource objects - they should be automatically processed with the resource method?